pkg/cmd/openshift-controller-manager/controller_manager.go

```diff
@@ -142,11 +135,11 @@ func newControllerContext(
 
 	// divide up the QPS since it re-used separately for every client
 	// TODO, eventually make this configurable individually in some way.
-	if privilegedLoopbackConfig.QPS > 0 {
-		privilegedLoopbackConfig.QPS = privilegedLoopbackConfig.QPS/10 + 1
+	if clientConfig.QPS > 0 {
+		clientConfig.QPS = clientConfig.QPS/10 + 1
```

Comment:
huh... mutating passed in config... do we need to copy locally first?

Reply:
> huh... mutating passed in config... do we need to copy locally first?

Pre-existing.  I don't think so since the rest is already fluffed out and the privileged one is used for list/watch.

Reply:
Updated to copy since it was easy and I didn't really have a reason not to.